src/html/boldOrItalic.js

```diff
@@ -67,13 +66,13 @@ function wrapping_bold_italic(tag, placeholders, chunks){
       } else {
         chunks.after = open + '>' + chunks.after;
       }
-    }
-    if (!closebounded(chunks, tag)) {
-      chunks.after = close + '>' + chunks.after;
-      chunks.before += open + '>';
+      }
+      if (!closebounded(chunks, tag)) {
+    	chunks.after = close + '>' + chunks.after;
+      	chunks.before += open + '>';
```

Comment:
Extra space at the beginning.

Reply:
Just now implementing the linting tool which should fix this as well